tasks/vlabTasks.sh

```diff
@@ -174,15 +182,12 @@ Task::check_version() {
   : @desc "Checks the current version"
 
   VERSION_CURRENT=$(cat VERSION)
-  VERSION_LATEST=$(cat VERSION)
-  #VERSION_LATEST=$(curl -s -m 2 https://github.com/Vivumlab/VivumLab/raw/master/VERSION)
-
-  function version_gt() { test "$(echo "$@" | tr " " "\n" | sort | head -n 1)" != "$1"; }
+  VERSION_LATEST=$(curl -s -m 2 https://raw.githubusercontent.com/Vivumlab/VivumLab/master/VERSION)
 
   colorize yellow "You currently have version: $VERSION_CURRENT"
   colorize green "The newest Version available is: $VERSION_LATEST"
 
-  if version_gt $VERSION_LATEST $VERSION_CURRENT; then
+  if version_check $VERSION_LATEST $VERSION_CURRENT; then
     colorize red "* You should update to version $VERSION_LATEST! *"
     colorize red " * Update at https://github.com/Vivumlab/VivumLab/-/releases *"
```

Comment:
This link does not work.
This fixes it.
```suggestion
    colorize red " * Update at https://github.com/Vivumlab/VivumLab/releases *"
```